geomesa-fs/geomesa-fs-datastore/src/main/scala/org/locationtech/geomesa/fs/data/FileSystemDataStore.scala

```diff
@@ -76,11 +76,13 @@ class FileSystemDataStore(
           }
           deprecated.getOrElse(true)
         }
+        val fileSize = sft.removeTargetFileSize()
```

Comment:
Out of curiosity, why remove the filesize from the SFT?  

I can imagine it is reasonable; just want to discuss it.

Reply:
this is a pattern we use for several of the fs config values. for convenience, we allow them to be set in the simple feature type, but then they end up getting persisted separately in the metadata class, so we don't want to keep 2 (potentially conflicting) values around. there's a comment at the top of the section that you can't see in the pr: https://github.com/locationtech/geomesa/blob/3273e45b6c06760bc7adcf9f1a4612a3315d8f68/geomesa-fs/geomesa-fs-datastore/src/main/scala/org/locationtech/geomesa/fs/data/FileSystemDataStore.scala#L62